lib/internal/Magento/Framework/Code/Test/Unit/GeneratorTest.php

```diff
@@ -156,7 +162,7 @@ public function testShouldNotGenerateVirtualType(): void
         $this->objectManagerConfigMock
             ->expects($this->once())
             ->method('getVirtualTypes')
-            ->willReturn([GeneratedClassFactory::class => GeneratedClassFactory::class]);
+            ->willReturn(['Magento\GeneratedClass\Factory' => 'Magento\GeneratedClass\Factory']);
```

Comment:
Why was it replaced with FQCN?

Reply:
Hello @ihor-sviziev ! Thank for your review. This changes needed to Static tests. The tests get an error with FQCN because Factory doesn't exist.

Reply:
Please add comment there